build/dockerfiles/Dockerfile

```diff
@@ -31,7 +31,7 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/lib64:/lib:/lib64
 COPY --from=build "$GOPATH/src/github.com/erda-project/erda/bin/${APP_NAME}" "/app/${APP_NAME}"
 COPY --from=build "$GOPATH/src/github.com/erda-project/erda/conf/${CONFIG_PATH}" "/app/conf/${CONFIG_PATH}"
 COPY --from=build "$GOPATH/src/github.com/erda-project/erda/pkg/erda-configs" "/app/erda-configs"
-COPY --from=build "$GOPATH/src/github.com/erda-project/erda/conf/common" "/app/common"
+COPY --from=build "$GOPATH/src/github.com/erda-project/erda/conf/common" "/app/conf/common"
```

Comment:
You need to adjust the place you use it.

Reply:
> You need to adjust the place you use it.

conf/common file shouldn't be root dictionary.